packaging.qmd

```diff
@@ -0,0 +1,52 @@
+# Packaging odin models
```

Comment:
This one isn't emerging into the HTML version yet?

Reply:
fixed, thanks